### ISSUE TITLE
Mib loading

### ISSUE THREAD

**Issue by ppranjivan** [splunk/splunk-connect-for-snmp#573]
Ability to add mibs locally currently does not exist. In air gapped environments pulling down external repos can be very arduous.

**Comment by M-Bates** [comment]
Second this. Should be a simple matter to add additional MIBS without having to interact with an external product.
Even the requirement of having to fork, merge etc to add MIBS is painful. All "production" ready SNMP systems have th eability to simply add a MIB.

**Comment by Tulfen** [comment]
It's coming

**Comment by omrozowicz-splunk** [comment]
Hey, this feature is already available as a beta in `1.8.6-beta.1`. 
Here is described how to configure: https://splunk.github.io/splunk-connect-for-snmp/develop/mib-request/#beta-use-mib-server-with-local-mibs 
And here how to upgrade to the beta version: https://splunk.github.io/splunk-connect-for-snmp/develop/upgrade/#upgrade-to-a-specific-version

It was already tested, but because of MIB specific we're not able to go through the all particular cases. We appreciate any feedback 😃

**Comment by omrozowicz-splunk** [comment]
UPDATE: the dependency build for an offline installation failed unexpectedly because of the Github who cannot handle files larger than 2GB (the new mibserver with compilation option is heavier than the previous one). 
It changes completely nothing for online installations where the only problem is proprietary mibs, but for air gapped I'd advise to wait a few days for the next beta with updated documentation of how to install it on an offline environment. 

If you don't want to wait, can pull mibserver:1.15.0 image by hand and import to the microk8s cluster, with:
```
docker pull ghcr.io/pysnmp/mibs/container:1.15.0 
docker save ghcr.io/pysnmp/mibs/container:1.15.0 > mibserver.tar 
```

I'll inform you when the version with fixed CI and the installation doc is ready
